plot_earthquakes.py

```diff
@@ -0,0 +1,119 @@
+from datetime import date
+import requests
+import json
+import numpy as np
+
+import matplotlib.pyplot as plt
+
+
+def get_data():
+    """Retrieve the data we will be working with."""
+    # With requests, we can ask the web service for the data.
+    # Can you understand the parameters we are passing here?
+    response = requests.get(
+        "http://earthquake.usgs.gov/fdsnws/event/1/query.geojson",
+        params={
+            'starttime': "2000-01-01",
+            "maxlatitude": "58.723",
+            "minlatitude": "50.008",
+            "maxlongitude": "1.67",
+            "minlongitude": "-9.756",
+            "minmagnitude": "1",
+            "endtime": "2018-10-11",
+            "orderby": "time-asc"}
+    )
+
+    # The response we get back is an object with several fields.
+    # The actual contents we care about are in its text field:
+    text = response.text
+
+    # Loading the text as a dictionary using json
+    return json.loads(text)
+
+
+def get_year(earthquake):
+    """Extract the year in which an earthquake happened."""
+    timestamp = earthquake['properties']['time']
+    # The time is given in a strange-looking but commonly-used format.
+    # To understand it, we can look at the documentation of the source data:
+    # https://earthquake.usgs.gov/data/comcat/index.php#time
+    # Fortunately, Python provides a way of interpreting this timestamp:
+    #  timestamp is in ms and fromtimestamp takes s so divide by 1000
+    year = date.fromtimestamp(timestamp/1000).year
+    return year
+
+
+def get_magnitude(earthquake):
+    """Retrive the magnitude of an earthquake item."""
+    return earthquake["properties"]["mag"]
+
+
+# This is function you may want to create to break down the computations,
+# although it is not necessary. You may also change it to something different.
+def get_magnitudes_per_year(earthquakes):
+    res = {}
+    for quake in earthquakes:
+        year = get_year(quake)
+        mag = get_magnitude(quake)
+        if year not in res.keys():
+            res[year] = [mag]
+        else:
+            res[year].append(mag)
+    """Retrieve the magnitudes of all the earthquakes in a given year.
+    
+    Returns a dictionary with years as keys, and lists of magnitudes as values.
+    """
```

Comment:
This "docstring" (a string documenting what the function does) should be at the top of the function, between the `def` line and the function body.